app/controllers/organizations/adoptable_pets_controller.rb

```diff
@@ -8,10 +8,9 @@ class Organizations::AdoptablePetsController < Organizations::BaseController
   helper_method :get_animals
 
   def index
-    @q = authorized_scope(Pet.includes(:adopter_applications, images_attachments: :blob),
-      with: Organizations::AdoptablePetPolicy).ransack(params[:q])
+    @q = authorized_scope(Pet.all, with: Organizations::AdoptablePetPolicy).ransack(params[:q])
```

Comment:
Do you think we should only return pets that are unadopted for this collection?

Reply:
Sure, I can do that. Okay if I add that to this PR?

Reply:
Added to this PR for now (see last commit).

Reply:
Yes in this PR is fine. Thanks.